salalib/axialmap.cpp

```diff
@@ -220,10 +218,10 @@ void ShapeGraph::outputNet(std::ostream& netfile) const
    }
 }
 
-typedef pvector<IntPair> IntPairVector;
+typedef std::vector<IntPair> IntPairVector;
 
 // n.b., translate radius list before entry
-bool ShapeGraph::integrate(Communicator *comm, const pvecint& radius_list, bool choice, bool local, bool fulloutput, int weighting_col, bool simple_version)
+bool ShapeGraph::integrate(Communicator *comm, const std::set<int>& radius_list, bool choice, bool local, bool fulloutput, int weighting_col, bool simple_version)

```

Comment:
A `set` shouldn't be called `radius_list` - can you rename it to `radius_set`, please?